doc/modules/cross_validation.rst

```diff
@@ -728,6 +728,10 @@ to shuffle the data indices before splitting them. Note that:
 * To ensure results are repeatable (*on the same platform*), use a fixed value
   for ``random_state``.
 
+The randomized CV splitters may return different results for each call of
```

Comment:
This is described in less clear terms in the preceding bullet point. Please merge them.